src/test/java/org/folio/rest/support/matchers/CollectionMatchers.java

```diff
@@ -0,0 +1,79 @@
+package org.folio.rest.support.matchers;
+
+import static org.hamcrest.core.AllOf.allOf;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import io.vertx.core.json.JsonObject;
+
+public final class CollectionMatchers {
```

Comment:
Was this class sourced from somewhere else?

Reply:
No, it is removed now.